src/core/logical_layers/utils/logicalLayerFabric.ts

```diff
@@ -75,7 +76,8 @@ export function createLogicalLayerAtom(
       },
     ) => {
       const actions: Action[] = [];
-      const map = getUnlistedState<ApplicationMap | null>(currentMapAtom);
+      const map =
```

Comment:
Mm, this is not at all how the map instance was supposed to be used.
When I designed the structure I assumed that we always have only one card, but here we have the existence of two cards at the same time. I'm not sure how this would work in other places where we still refer to currentMap

Reply:
as we discussed I'll create an issue about refactoring and then we'll refactor this place